frontend-web/webclient/app/UCloud/JobsApi.tsx

```diff
@@ -304,6 +306,22 @@ class JobApi extends ResourceApi<Job, ProductCompute, JobSpecification, JobUpdat
             shortcut: ShortcutKey.P
         }];
 
+        if (hasFeature(Feature.JOB_RENAME)) {
+            ourOps.push({
+                enabled(selected) {
+                    // TODO(Jonas): Should this work for every state of a job?
+                    // What about apps like syncthing?
```

Comment:
```suggestion
                    // NOTE(Dan): This should work regardless of job and job state. Even for provider registered 
                    // jobs this shouldn't be an issue since this is just a name used by the end-user.
```